React-Homework02

JohnYazji opened the pull request.

JohnYazji committed:
Add files via upload

braedongough commented on react/week2/src/index.js:
```diff
@@ -0,0 +1,106 @@
+ReactDOM.render(
+  <React.StrictMode>
+    <h1> My todo list</h1>
+    <App />
```
`App` isn't imported or declared anywhere in this file. It also doesn't look like you're using your `TodoList` component anywhere. Did you make sure to run this and see if it actually works?

braedongough reviewed (changes requested):
The code looks like its functionally correct, however you're missing your package.json here, so I couldn't run it locally on my machine - was that intentional? 

Also, it looks like you're using a component `App` that doesn't exist and you don't use the `TodoList` component at all, which makes me think this might not have been tested to work. Do you get any errors when running locally?

JohnYazji commented:
> The code looks like its functionally correct, however you're missing your package.json here, so I couldn't run it locally on my machine - was that intentional?

> I tried to upload all my-app files into Github, but I couldn't. So I only chose the src files. That why I missed the "package.json" Do you need me to try to upload it here again?

JohnYazji committed:
Update index.js

I have made a few little changes on the code, but I still not be able to run the code properly, even though I think it has been coded right. kindly advise. I got this message on the browser page:
./src/index.js
  Line 97:9:  Parsing error: Unexpected token
     95 | 
     96 | 
>   97 | ReactDOM.render(
          |                  ^
     98 |   <React.StrictMode>
     99 |     <TodoListItem />
   100 |   </React.StrictMode>,
